.pipelines/vhd-builder-windows.yaml

```diff
@@ -1,12 +1,27 @@
+# Required pipeline variables:
+# - AZURE_LOCATION - Azure region to create resources in
+# - AZURE_RESOURCE_GROUP_NAME - Resource group to create resources in
+# - BUILD_POOL - Azure DevOps build pool to use
+# - CLIENT_ID - Service principal ID
+# - CLIENT_SECRET - Service principal secret
+# - COPY_VHD - Set to 'True' if VHD should be copied to classic storage account in preperation for publishing
+# - DEIS_GO_DEV_IMAGE - Dev container image ID
+# - PACKER_VM_SIZE - VM Size to be used during packer build operation
+# - STORAGE_ACCT_BLOB_URL - Url to blob storage container used to copy image to in preperation for publishing
+# - STORAGE_ACCT_SAS_TOKEN - SAS token with permission to write to CLASSIC_BLOB container
+# - SUBSCRIPTION_ID - Subscription ID used by the pipeline
+# - TENANT_ID - Tenant Id used for service principal login
+
 trigger: none
+pr: none
 
 jobs:
 - job: build_vhd_windows
   timeoutInMinutes: 120
   strategy:
     maxParallel: 0
   pool:
-    name: Hosted Ubuntu 1604
+    name: $(BUILD_POOL)

```

Comment:
the only thing i notice is that this is a little weird but /shrug not a big deal

Reply:
I needed to move away from using hosted agents because the jobs went past the 60 minute time limit after adding the E2E tests.
I made it a pipeline variable so the .yaml could be reusable across DevOps accounts.